src/fetch.js

```diff
@@ -0,0 +1,17 @@
+const fetchStarWarsData = async (type) => {
```

Comment:
You can add 'export' at the beggining of the first line: export const fetchStarWarsData = async (type) => {...